schemas/get-worker-type-last-modified.yml

```diff
@@ -8,7 +8,9 @@ properties:
     type: string
     description: |
       The ID of the workerType
-    pattern: ^[A-Za-z0-9+/=_-]{1,22}$
```

Comment:
Note that the pattern has changed here, but we use the same newly-defined `identifier-pattern` (`"^([a-zA-Z0-9-_]*)$"`) in the queue, so that's OK.